joeynmt/training.py

```diff
@@ -391,7 +398,7 @@ def train_and_validate(self, train_data: Dataset, valid_data: Dataset) \
                     elif self.early_stopping_metric in ["ppl", "perplexity"]:
                         ckpt_score = valid_ppl
                     else:
-                        ckpt_score = valid_score
+                        ckpt_score = valid_scores[0]
```

Comment:
Okay, so you're already using the first metric as the early stopping one. Good :)